tests/feinterface/stokes.output

```diff
@@ -0,0 +1,256 @@
+
+DEAL::FESystem<2>[FESystem<2>[FE_DGQ<2>(2)^2]-FE_DGQ<2>(1)]: degree=2 tensor_degree=2
+DEAL::  Now running with FESystem<2>[FESystem<2>[FE_DGQ<2>(2)^2]-FE_DGQ<2>(1)]
+DEAL::	Number of active cells: 16
+DEAL::	Number of degrees of freedom: 352 (288+64)
+DEAL:FGMRES::Starting value 13.
+DEAL:FGMRES:cg::Starting value 1.2e-33
+DEAL:FGMRES:cg::Convergence step 1 value 1.9e-49
+DEAL:FGMRES:cg::Starting value 1.0
+DEAL:FGMRES:cg::Convergence step 1 value 4.9e-15
+DEAL:FGMRES:cg::Starting value 1.0
+DEAL:FGMRES:cg::Convergence step 1 value 2.0e-16
+DEAL:FGMRES:cg::Starting value 7.5
+DEAL:FGMRES:cg::Convergence step 1 value 3.0e-14
```

Comment:
Can we avoid printing these numbers. They tend to differ for different compiler setups and we are probably only interested in errors here and not any solver information.